geode-for-redis/src/integrationTest/java/org/apache/geode/redis/internal/commands/executor/set/AbstractSDiffStoreIntegrationTest.java

```diff
@@ -66,6 +66,30 @@ public void sdiffstore_DifferentKeyType_returnsWrongTypeError() {
         .hasMessageContaining(ERROR_WRONG_TYPE);
   }
 
+  @Test
+  public void sdiffstore_withDifferentKeyTypeAndTwoSetKeys_returnsWrongTypeError() {
```

Comment:
This test name might be a little clearer as something like "sdiffstore_withNonSetKeyAsFirstSourceKey_returnsWrongTypeError"